pkg/storages/sh/folder.go

```diff
@@ -45,49 +43,17 @@ func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder,
 		return nil, err
 	}
 
-	user := settings[Username]
```

Comment:
Ok, why we can't instead configure it by creating some config struct (`viper.New()`) with SSH options and init a folder? 

It will allow to use the existing storages implementation without any modification.

You can use this code as a reference
https://github.com/wal-g/wal-g/blob/3b3ec011f2c17bc46397046a83e6cb2671261d87/internal/config.go#L702-L720

Reply:
I don't quite understand. This code, if dive into, works with the code presented above.